### ISSUE TITLE
Editor crash if set a vehicle actor as disabled

### ISSUE THREAD

**Issue by RuanLucasGD** [FlaxEngine/FlaxEngine#1351]
**Issue description:**
if you create a vehicle and disable the actor, you cannot play scene, editor crash. If you has a scene saved and close flax you cannot reopen project.

**Minimal reproduction project:**
[Car Test.zip](https://github.com/FlaxEngine/FlaxEngine/files/12455133/Car.Test.zip)

**Flax version:**
1.6

**Comment by Tryibion** [comment]
Here is the error:

```
Fatal error. System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
Repeat 2 times:
--------------------------------
   at FlaxEngine.Actor.<Internal_SetIsActive>g____PInvoke|50_0(IntPtr, Byte)
--------------------------------
   at FlaxEngine.Actor.Internal_SetIsActive(IntPtr, Boolean)
   at FlaxEngine.Actor.set_IsActive(Boolean)
   at DynamicClass.InvokeStub_Actor.set_IsActive(System.Object, System.Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(System.Object, IntPtr*, System.Reflection.BindingFlags)
   at System.Reflection.RuntimeMethodInfo.InvokeOneParameter(System.Object, System.Reflection.BindingFlags, System.Reflection.Binder, System.Object, System.Globalization.CultureInfo)
   at System.Reflection.RuntimePropertyInfo.SetValue(System.Object, System.Object, System.Reflection.BindingFlags, System.Reflection.Binder, System.Object[], System.Globalization.CultureInfo)
   at System.Reflection.PropertyInfo.SetValue(System.Object, System.Object)
   at FlaxEditor.Scripting.ScriptMemberInfo.SetValue(System.Object, System.Object)
   at FlaxEditor.CustomEditors.ValueContainer.Set(FlaxEditor.CustomEditors.ValueContainer, System.Object)
   at FlaxEditor.CustomEditors.CustomEditor.SynchronizeValue(System.Object)
   at FlaxEditor.CustomEditors.CustomEditor.RefreshInternal()
   at FlaxEditor.CustomEditors.CustomEditor.RefreshRootChild()
   at FlaxEditor.CustomEditors.CustomEditor.RefreshRoot()
   at FlaxEditor.CustomEditors.SyncPointEditor.Refresh()
   at FlaxEditor.CustomEditors.SyncPointEditor.RefreshInternal()
   at FlaxEditor.CustomEditors.CustomEditorPresenter.Update()
   at FlaxEditor.CustomEditors.CustomEditorPresenter+PresenterPanel.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.ContainerControl.Update(Single)
   at FlaxEngine.GUI.RootControl.Update(Single)
   at FlaxEngine.Window.Internal_OnUpdate(Single)
   at FlaxEngine.Interop.NativeInterop+Invoker+InvokerNoRet1`2[[System.__Canon, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.Single, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MarshalAndInvoke(System.Object, FlaxEngine.Interop.ManagedHandle, IntPtr)
   at FlaxEngine.Interop.NativeInterop.InvokeMethod(FlaxEngine.Interop.ManagedHandle, FlaxEngine.Interop.ManagedHandle, IntPtr, IntPtr)
```

**Comment by mafiesto4** [comment]
Fixed in 639149b63b4a609802e984137fbb4ecd3728543f.
